apps/web/components/seo/JsonLd.tsx

```
@@ -0,0 +1,199 @@
+    sameAs,
+    contactPoint: {
+      '@type': 'ContactPoint',
+      email: 'info@carsi.com.au',
+      contactType: 'customer service',
+    },
+  };
+
+  return (
+    <script
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{ __html: JSON.stringify(schema) }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)